buildscripts/build.sh

```diff
@@ -33,7 +39,34 @@ go_build() {
     package=$(go list -f '{{.ImportPath}}')
     echo -n "-->"
     printf "%15s:%s\n" "${osarch}" "${package}"
-    GO15VENDOREXPERIMENT=1 GOOS=$os GOARCH=$arch go build --ldflags "${LDFLAGS}" -o $release_str/$os-$arch/$(basename $package).$release_tag
+
+    # Release binary name
+    release_bin="$release_str/$os-$arch/$(basename $package).$release_tag"
+    # Release binary downloadable name
+    release_real_bin="$release_str/$os-$arch/$(basename $package)"
+    # Release shasum name
+    release_shasum="$release_str/$os-$arch/$(basename $package).shasum"
+    # Release zip file.
+    release_real_zip="$release_str/$os-$arch/$(basename $package).zip"
+    release_real_gz="$release_str/$os-$arch/$(basename $package).gz"
+
+    # Go build to build the binary.
+    GO15VENDOREXPERIMENT=1 GOOS=$os GOARCH=$arch go build --ldflags "${LDFLAGS}" -o $release_bin
```

Comment:
-    GO15VENDOREXPERIMENT=1 GOOS=$os GOARCH=$arch go build --ldflags "${LDFLAGS}" -o $release_bin
+   go build GO15VENDOREXPERIMENT=1 GOOS=$os GOARCH=$arch --ldflags "${LDFLAGS}" -o $release_bin
is more readable.